client/components/domains/registrant-extra-info/uk-form.jsx

```diff
@@ -0,0 +1,198 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { camelCase, difference, includes, isEmpty, keys, map, pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getContactDetailsCache } from 'state/selectors';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+import { updateContactDetailsCache } from 'state/domains/management/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+
+const defaultValues = {
+	registrantType: 'IND',
+};
+
+class RegistrantExtraInfoUkForm extends React.PureComponent {
+	static propTypes = {
+		contactDetails: PropTypes.object.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+		const { translate } = props;
+		const registrantTypes = {
+			IND: translate( 'Individual' ),
+			FIND: translate( 'Foreign Individual' ),
+			STRA: translate( 'UK Sole Trader', {
+				comment: 'Refers to UK legal concept of self-employment/sole proprietorship',
+			} ),
+			PTNR: translate( 'UK Partnership' ),
+			LLP: translate( 'UK Limited Liability Partnership' ),
+			CRC: translate( 'UK Corporation by Royal Charter' ),
+			FCORP: translate( 'Non-UK Corporation' ),
+			IP: translate( 'UK Industrial/Provident Registered Company' ),
+			LTD: translate( 'UK Limited Company' ),
+			PLC: translate( 'UK Public Limited Company' ),
+			SCH: translate( 'UK School' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'School', { context: 'Theme Showcase term name'} )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/cctld-uk)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).